extensions/msal-node-extensions/package.json

```diff
@@ -68,5 +68,8 @@
   },
   "bundledDependencies": [
     "@azure/msal-common"
+  ],
+  "bundleDependencies": [
```

Comment:
Looks like "bundledDependencies" is already here. We should stick to one or the other